pkg/workloads/console/integration/integration_test.go

```diff
@@ -166,8 +179,38 @@ var _ = Describe("Console", func() {
 		// test that here.
 	})
 
+	Describe("Invalid template specifications", func() {
+		It("Emits a warning if there are multiple containers present", func() {
+			By("Creating a console with a template with more than 1 container")
+
+			tpl := buildConsoleTemplate(namespace)
+			// Duplicate the container
+			tpl.Spec.Template.Spec.Containers = append(
+				tpl.Spec.Template.Spec.Containers,
+				tpl.Spec.Template.Spec.Containers[0],
+			)
+			tpl.Spec.Template.Spec.Containers[1].Name = "sidecar"
+
+			createTemplate(tpl)
+			createConsole(1)
+
+			waitForSuccessfulReconcile(2)
+
+			// By("Expect a warning event")
```

Comment:
Ticket link here too

Reply:
extracted to another PR for future reference